figures/S14/data/apoptosis_results.csv

```diff
@@ -1,37 +1,38 @@
 Search query: apoptosis
```

Comment:
Have you done anything to the data collection to refine your search? Earlier it seems like they got an extra publication, I don't know if you're aware of that. (e.g.2011, 2009, 2005, 2004 and 1993)

Reply:
Who is they and what extra publication?